Remove defined of __{DATE,TIMESTAMP,TIME}__
Intel groks on using these on cmd line.

Davide Vanzo, 20190213
diff -ru bazel.orig/src/test/java/com/google/devtools/build/lib/analysis/mock/MOCK_CROSSTOOL bazel/src/test/java/com/google/devtools/build/lib/analysis/mock/MOCK_CROSSTOOL
--- bazel.orig/src/test/java/com/google/devtools/build/lib/analysis/mock/MOCK_CROSSTOOL	1980-01-01 00:00:00.000000000 -0600
+++ bazel/src/test/java/com/google/devtools/build/lib/analysis/mock/MOCK_CROSSTOOL	2019-02-13 14:37:29.799413169 -0600
@@ -87,9 +87,6 @@
   # Make C++ compilation deterministic. Use linkstamping instead of these
   # compiler symbols.
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
 
   # Security hardening on by default.
   # Conservative choice; -D_FORTIFY_SOURCE=2 may be unsafe in some cases.
diff -ru bazel.orig/src/test/shell/bazel/testdata/bazel_toolchain_test_data/tools/arm_compiler/CROSSTOOL bazel/src/test/shell/bazel/testdata/bazel_toolchain_test_data/tools/arm_compiler/CROSSTOOL
--- bazel.orig/src/test/shell/bazel/testdata/bazel_toolchain_test_data/tools/arm_compiler/CROSSTOOL	1980-01-01 00:00:00.000000000 -0600
+++ bazel/src/test/shell/bazel/testdata/bazel_toolchain_test_data/tools/arm_compiler/CROSSTOOL	2019-02-13 14:41:00.643419003 -0600
@@ -99,9 +99,6 @@
   # Make C++ compilation deterministic. Use linkstamping instead of these
   # compiler symbols.
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
 
   # Security hardening on by default.
   # Conservative choice; -D_FORTIFY_SOURCE=2 may be unsafe in some cases.
@@ -202,9 +199,6 @@
   target_system_name: "local"
   unfiltered_cxx_flag: "-fno-canonical-system-headers"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   tool_path {name: "ar" path: "/usr/bin/ar" }
   tool_path {name: "cpp" path: "/usr/bin/cpp" }
   tool_path {name: "dwp" path: "/usr/bin/dwp" }
diff -ru bazel.orig/tools/cpp/CROSSTOOL bazel/tools/cpp/CROSSTOOL
--- bazel.orig/tools/cpp/CROSSTOOL	1980-01-01 00:00:00.000000000 -0600
+++ bazel/tools/cpp/CROSSTOOL	2019-02-13 14:42:51.283422065 -0600
@@ -141,9 +141,6 @@
   # Make C++ compilation deterministic. Use linkstamping instead of these
   # compiler symbols.
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
 
   # Security hardening on by default.
   # Conservative choice; -D_FORTIFY_SOURCE=2 may be unsafe in some cases.
@@ -247,9 +244,6 @@
   # Make C++ compilation deterministic. Use linkstamping instead of these
   # compiler symbols.
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
 
   # Security hardening on by default.
   # Conservative choice; -D_FORTIFY_SOURCE=2 may be unsafe in some cases.
@@ -352,9 +346,6 @@
   # Make C++ compilation deterministic. Use linkstamping instead of these
   # compiler symbols.
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
 
   # Security hardening on by default.
   # Conservative choice; -D_FORTIFY_SOURCE=2 may be unsafe in some cases.
diff -ru bazel.orig/tools/cpp/unix_cc_configure.bzl bazel/tools/cpp/unix_cc_configure.bzl
--- bazel.orig/tools/cpp/unix_cc_configure.bzl	1980-01-01 00:00:00.000000000 -0600
+++ bazel/tools/cpp/unix_cc_configure.bzl	2019-02-13 14:43:29.863423132 -0600
@@ -249,9 +249,6 @@
               # Make C++ compilation deterministic. Use linkstamping instead of these
               # compiler symbols.
               "-Wno-builtin-macro-redefined",
-              "-D__DATE__=\\\"redacted\\\"",
-              "-D__TIMESTAMP__=\\\"redacted\\\"",
-              "-D__TIME__=\\\"redacted\\\""
           ],
       "compiler_flag": [
           # Security hardening requires optimization.
diff -ru bazel.orig/tools/osx/crosstool/CROSSTOOL.tpl bazel/tools/osx/crosstool/CROSSTOOL.tpl
--- bazel.orig/tools/osx/crosstool/CROSSTOOL.tpl	1980-01-01 00:00:00.000000000 -0600
+++ bazel/tools/osx/crosstool/CROSSTOOL.tpl	2019-02-13 14:44:55.715425508 -0600
@@ -153,9 +153,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   default_python_version: "python2.7"
   feature {
     name: "fastbuild"
@@ -1754,9 +1751,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-target"
   unfiltered_cxx_flag: "x86_64-apple-ios"
   default_python_version: "python2.7"
@@ -3373,9 +3367,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-target"
   unfiltered_cxx_flag: "i386-apple-watchos"
   default_python_version: "python2.7"
@@ -4995,9 +4986,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-target"
   unfiltered_cxx_flag: "x86_64-apple-tvos"
   default_python_version: "python2.7"
@@ -6644,9 +6632,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-target"
   unfiltered_cxx_flag: "i386-apple-ios"
   default_python_version: "python2.7"
@@ -8263,9 +8248,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-target"
   unfiltered_cxx_flag: "armv7-apple-ios"
   default_python_version: "python2.7"
@@ -9870,9 +9852,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-target"
   unfiltered_cxx_flag: "armv7k-apple-watchos"
   default_python_version: "python2.7"
@@ -11480,9 +11459,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-target"
   unfiltered_cxx_flag: "arm64-apple-tvos"
   default_python_version: "python2.7"
@@ -13117,9 +13093,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-target"
   unfiltered_cxx_flag: "arm64-apple-ios"
   default_python_version: "python2.7"
@@ -14724,9 +14697,6 @@
   builtin_sysroot: ""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
   unfiltered_cxx_flag: "-Wno-builtin-macro-redefined"
-  unfiltered_cxx_flag: "-D__DATE__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIMESTAMP__=\"redacted\""
-  unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   supports_normalizing_ar: false
   supports_start_end_lib: false
   default_python_version: "python2.7"
